feature-libs/cart/base/components/clear-cart/clear-cart-button/clear-cart.component.html

```diff
@@ -0,0 +1,11 @@
+<div *ngIf="cart$ | async as cart" class="clear-cart-wrapper">
```

Comment:
We could consider subscribing in a container tag and hide the div if the cart id empty.  Let's discuss.